analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnchangedLocalVariablesShouldBeConst.ToFix.cs

```diff
@@ -0,0 +1,35 @@
+﻿enum Colors { Red, Blue, Green };
```

Comment:
For the namespace handling:
```suggestion
using System;

﻿﻿enum Colors { Red, Blue, Green };
```